README.md

```diff
@@ -1,5 +1,5 @@
 # AWS Service Catalog Engine for Terraform Cloud
-The AWS Service Catalog Engine for Terraform Cloud (TFC-RE) is an integration between AWS Service Catalog and Terraform Cloud that allows users to provision Service Catalog products using TFC. This integration gives administrators governance and visibility into their Terraform workloads, and allows Service Catalog administrators to delegate cloud resource provisioning responsibilities to users within their organizations.
+The AWS Service Catalog Engine for Terraform Cloud (TFC-RE) is an integration between AWS Service Catalog and Terraform Cloud/Enterprise that allows users to provision Service Catalog products using TFC. This integration gives administrators governance and visibility into their Terraform workloads, and allows Service Catalog administrators to delegate cloud resource provisioning responsibilities to users within their organizations.
```

Comment:
Quick suggestion: Should we add "/E" to highlight Enterprise support?:
```suggestion
The AWS Service Catalog Engine for Terraform Cloud (TFC-RE) is an integration between AWS Service Catalog and Terraform Cloud/Enterprise that allows users to provision Service Catalog products using TFC/E. This integration gives administrators governance and visibility into their Terraform workloads, and allows Service Catalog administrators to delegate cloud resource provisioning responsibilities to users within their organizations.
```

Reply:
🤔 Good catch... I think I like the idea of avoiding the abbreviations altogether, but also doing what you suggest here

Reply:
> I think I like the idea of avoiding the abbreviations altogether, but also doing what you suggest here

I agree - the struggle! 

> how does that look to you?

I think it looks great! Since we already use abbreviations, this edit is consistent. 🌟